pkg/resource/deploy/plan_executor.go

```diff
@@ -303,39 +312,54 @@ func (pe *planExecutor) refresh(callerCtx context.Context, opts Options, preview
 	stepExec.SignalCompletion()
 	stepExec.WaitForCompletion()
 
-	// Rebuild this plan's map of old resources and dependency graph, stripping out any deleted resources and repairing
-	// dependency lists as necessary. Note that this updates the base snapshot _in memory_, so it is critical that any
-	// components that use the snapshot refer to the same instance and avoid reading it concurrently with this rebuild.
+	// Rebuild this plan's map of old resources and dependency graph, stripping out any deleted
+	// resources and repairing dependency lists as necessary. Note that this updates the base
+	// snapshot _in memory_, so it is critical that any components that use the snapshot refer to
+	// the same instance and avoid reading it concurrently with this rebuild.
```

Comment:
just wrapping for this comment.

Reply:
Why? It's already wrapped at 120 as per the guidelines, right?